### DISCUSSION TITLE
Cache busting best practice

### DISCUSSION THREAD

**Discussion by konojunya** [vercel/next.js#18985] [Show and tell]
The default Cache-Control setting for public in Next.js is `public, max-age: 0, must-revalidate`.

https://vercel.com/docs/edge-network/caching?query=static#static-files

I just want to take a revving cache strategy with something like BuildID with a longer Cache.

https://developer.mozilla.org/en-US/docs/Web/HTTP/Caching

For example, the implementation of the page under /_next.

https://nextjs.org/blog/next-9-5#persistent-caching-for-page-bundles

If you start the server with `next start` , it's running on next's server, so it seems possible to set it freely (or incorporate it into Next's design), but do `next export’  and use Firebase or S3.  This is not the case if you do hosting.  So, is there any best practice on how to keep the Cache for a long time and disable it every time you build?

**Comment by Baryonae** [comment]
hi, can you provide some more info about cache busting? 
i dont wanna keep my cache during my web runtime, it doesnt let me update the data from supabase, pls help if possible...
